File: web-forms/src/store/evaluation/nutricional.js
import { defineStore } from 'pinia';

export const useNutricionalStore = defineStore('nutricional', {
    state: () => ({
        nutricional: {
    
            alimentacao_oral: false,
            alimentacao_enteral: false,
            alimentacao_parenteral: false,
            sonda: false,
            botton: false,
            suplemento: false,

            via_enteral: null,
            via_parenteral: null,
            adaptador_sonda: null,
            restricao_alimentar: '',
            tipo_dieta: null,
            forma_administracao: null,
            marca_bomba: null,

            data_ultima_troca: null,
            gavando: '',
            volume_diario: '',
            qual_suplemento: ''
        }
    }),
    actions: {
        clean() {
            this.nutricional = {
                alimentacao_oral: false,
                alimentacao_enteral: false,
                alimentacao_parenteral: false,
                sonda: false,
                botton: false,
                suplemento: false,
                via_enteral: null,
                via_parenteral: null,
                adaptador_sonda: null,
                restricao_alimentar: '',
                tipo_dieta: null,
                forma_administracao: null,
                marca_bomba: null,
                data_ultima_troca: null,
                gavando: '',
                volume_diario: '',
                qual_suplemento: ''
            };
        }
    }
});

File: backend/internal/avaliation/domain/nutricional.go
package domain

import "time"

type Nutricional struct {
	AlimentacaoOral       bool      `json:"alimentacao_oral"`
	AlimentacaoEnteral    bool      `json:"alimentacao_enteral"`
	AlimentacaoParenteral bool      `json:"alimentacao_parenteral"`
	Sonda                 bool      `json:"sonda"`
	Botton                bool      `json:"botton"`
	Suplemento            bool      `json:"suplemento"`
	ViaEnteral            string    `json:"via_enteral"`
	ViaParenteral         string    `json:"via_parenteral"`
	AdaptadorSonda        string    `json:"adaptador_sonda"`
	RestricaoAlimentar    string    `json:"restricao_alimentar"`
	TipoDieta             string    `json:"tipo_dieta"`
	FormaAdministracao    string    `json:"forma_administracao"`
	MarcaBomba            string    `json:"marca_bomba"`
	DataUltimaTroca       time.Time `json:"data_ultima_troca"`
	Gavando               string    `json:"gavando"`
	VolumeDiario          string    `json:"volume_diario"`
	QualSuplemento        string    `json:"qual_suplemento"`
}
